3-Authorization-II/2-call-api-b2c/README.md

```diff
@@ -197,26 +230,68 @@ To setup your B2C user-flows, do the following:
 
 Were we successful in addressing your learning objective? Consider taking a moment to [share your experience with us](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR73pcsbpbxNJuZCMKN0lURpUMlRHSkc5U1NLUkxFNEtVN0dEOTFNQkdTWiQlQCN0PWcu).
 
-## About the code
+## Troubleshooting
 
-### Token Validation
+Use [Stack Overflow](http://stackoverflow.com/questions/tagged/msal) to get support from the community. Ask your questions on Stack Overflow first and browse existing issues to see if someone has asked your question before. Make sure that your questions or comments are tagged with [`azure-active-directory-b2c` `react` `ms-identity` `adal` `msal`].
 
-[passport-azure-ad](https://github.com/AzureAD/passport-azure-ad) validates the token against the `issuer`, `scope` and `audience` claims (defined in `BearerStrategy` constructor) using the `passport.authenticate()` API:
+If you find a bug in the sample, raise the issue on [GitHub Issues](../../../../issues).
 
-```javascript
-    app.get('/api', passport.authenticate('oauth-bearer', { session: false }),
-        (req, res) => {
-            console.log('Validated claims: ', req.authInfo);
-    );
-```
+## About the code
```

Comment:
Can we make sure this About the code section matches to what is in 3-1 (except the parts that are not relevant e.g. idtyp etc.)

Reply:
done